### PR TITLE
For #311: Added ItemCreateView

### PR DESCRIPTION
**Pull request by paulodamaso**
For #311: Added `ItemCreateView`
- Added `ItemCreateView` implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#438](https://github.com/timelesslounge/timelessis/issues/438) is now in scope, role is `REV`

**Comment by 0crat** [comment]
Job `gh:timelesslounge/timelessis#438` already assigned to @vladarefiev, can't assign to @vladarefiev

**Comment by 0crat** [comment]
This pull request [#438](https://github.com/timelesslounge/timelessis/issues/438) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/117f8f9b-103b-466c-831b-9db858c49c30); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/view/items/test_items.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def test_list(client):
 
 def test_create(client):
     """ Test create is okay """
-    assert client.get("/items/create").status_code == HTTPStatus.OK
+    assert client.post("/items/create").status_code == HTTPStatus.OK
```
Comment (vladarefiev) [inline tests/view/items/test_items.py]: @paulodamaso let's improve test to test that item is really creating during the request. Now test is poor.

**Comment by vladarefiev** [comment]
@paulodamaso I like it, just only one comment.

File: tests/view/items/test_items.py (hunk 1 above)
Comment (vladarefiev) [inline tests/view/items/test_items.py]: @paulodamaso It's interesting, I thought here will be `http 302`. Let's test it.

**Comment by paulodamaso** [comment]
@vladarefiev Corrections made, please take a look

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 17 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#438](https://github.com/timelesslounge/timelessis/issues/438) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Confirm

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Pull request merged**